hw2/04-background.html

```diff
@@ -1,29 +1,30 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>04 - Background</title>
-    <link
-      rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
-    <script defer src="04-background.js"></script>
-  </head>
-  <body>
-    <main>
-      <h1>Exercise 04 - Background</h1>
-      <p>
-        The background color will change every few seconds (as indicated below).
-        You can stop this behavior and you can also change how often the
-        background color changes.
-      </p>
-      <div>
-        <label>Background change interval (seconds): </label>
-        <input />
-      </div>
-      <div>
-        <input />
-      </div>
-    </main>
-  </body>
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>04 - Background</title>
+		<link
+			rel="stylesheet"
+			href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+		/>
+		<script defer src="04-background.js"></script>
+	</head>
+	<body class="container" id="body" style="background-color: black">
+		<main class="bg-light mx-auto my-5 w-50">
+			<div class="p-3">
+				<h1>Exercise 04 - Background</h1>
+				<p>
+					The background color will change every few seconds (as indicated
+					below). You can stop this behavior and you can also change how often
+					the background color changes.
+				</p>
+				<div>
+					<label>Background change interval (seconds): </label>
+					<input id="timer" type="number" value="1" />
```

Comment:
WAVE accessibility indicates that there isn't the input doesn't have a corresponding label with it.